piaware-ble-connect/led.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+
+# Helper functions to control LEDs on the Pi
+
+import os
+
+def blink(led_num):
+    cmd = f"echo timer | sudo tee /sys/class/leds/{led_num}/trigger 1>/dev/null"
```

Comment:
It'd probably be cleaner to use `subprocess.run` here (and avoid the `echo` altogether)